packages/tldraw/src/lib/ui/components/Toolbar/DefaultToolbarContent.tsx

```diff
@@ -0,0 +1,52 @@
+import { useReadonly } from '../../hooks/useReadonly'
+import { useTools } from '../../hooks/useTools'
+import { TldrawUiMenuItem } from '../menus/TldrawUiMenuItem'
+
+/** @public */
+export function DefaultToolbarContent() {
```

Comment:
Where does this get used?

Reply:
Not used yet. I'll bin it until we get to the toolbar.